debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -364,21 +365,58 @@ public Lsn getFromLsn() {
     }
 
     public Set<SqlServerChangeTable> listOfChangeTables(String databaseName) throws SQLException {
```

Comment:
Let's return `List` here (same below) and also change the name, e.g. `getChangeTables()`. The current mismatch between name and actual return type is a bit confusing. Returning `Set` isn't useful either, given `SqlServerChangeTable` doesn't implement equals/hashCode.

Reply:
There seems to be a mix of the terminology used around capture instances. They are referred to as "change tables" (framework-level term) but also "CDC tables" and "CDC-enabled tables". The latter is especially incorrect because there may be multiple capture instances per CDC-enabled table.

I renamed a few more symbols to use the "capture instance" term which corresponds to the term in SQL Server.

Reply:
Yes, totally agreed that we don't use the terminology consistently here. So does SQL Server docs refer to it as this:

* "CDC-enabled table": the actual table in the user's data model from which changes should be captured
* "Capture instance": one (out of potentially multiple) special "CDC tables" for a CDC-enabled table, which we scan to obtain the changes

? If so, we should do use this consistently in code and docs. +1 for cleaning that up separately.

Reply:
I think it needs more thorough research. I got confused.

The documentation [uses](https://docs.microsoft.com/en-us/sql/relational-databases/system-tables/cdc-capture-instance-ct-transact-sql?view=sql-server-ver15) both the "change table" and "capture instance" terms, even in the same word:
> `cdc.<capture_instance>_CT (Transact-SQL)`.

Let me roll back the changes that replace "change table" with "capture instance" and only leave the ones that clearly make sense. E.g. `GET_LIST_OF_CDC_ENABLED_TABLES` which returns change tables from the `cdc.change_tables` table should be named `GET_LIST_OF_CHANGE_TABLES`.

Reply:
So what do you think, should we log a follow-up issue for using terminology more consistently here? Or is it ok now actually? It seems there's "source table" and "CDC-enabled table" vs. "change table" and "capture instance". If we don't use one of the former for referring to the latter (seeing you changed that in one place, thx), it might be ok actually?

Reply:
I believe it is okay now. At least there's no inconsistency in the code affected by this patch. Filing an issue and identifying the remaining problems would require extra effort at this point.

Reply:
> I believe it is okay now

Ok, cool.

> "CDC-enabled table" makes sense when we try to distinguish the regular tables by the fact whether they are CDC-enabled or not.

Makes sense. Ha, there might even be a CDC-enabled source table, LOL.

In any case, will merge this one once CI has passed.